app/assets/stylesheets/components/_filter-section.scss

```diff
@@ -0,0 +1,56 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-c-filter-section {
+  border-bottom: 1px solid $govuk-border-colour;
+  margin: govuk-spacing(4) 0;
+
+  &:last-child {
+    border-bottom: 0;
+    margin-bottom: 0; 
+  }
+
+  &:first-child {
+    margin-top: 0; 
+  }
+
+  &[open] .app-c-filter-section__summary::before {
+    transform: translateY(70%) rotate(-45deg);
+  }
+}
+
+.app-c-filter-section__summary {
+  display: flex;
+
+  &::before {
+    border-style: solid;
+    border-width: 2px 2px 0 0;
+    content: '';
+    display: inline-block;
+    height: govuk-spacing(2);
+    left: 0;
+    position: relative;
+    vertical-align: middle;
+    width: govuk-spacing(2);
+    border-color: $govuk-text-colour;
+    clip-path: unset;
+    transform: translateY(30%) rotate(135deg);
+  }
+
+  &-content {
```

Comment:
There's a preference here to have BEM blocks and modifiers all separately in the SCSS file (so they're easier to search for in the code) instead of using parent modifier like this 💡